examples/example_batch_detector.cpp

```diff
@@ -3,29 +3,40 @@
 
 #include "trace.hpp"
 #include <gflags/gflags.h>
-#include <opencv2/opencv.hpp>
+#include <iostream>
 
 #include "pose_detector.h"
 #include "utils.hpp"
 
 // Model flags
-DEFINE_string(model_file, "vgg.uff", "Path to uff model.");
-DEFINE_int32(input_height, 368, "Height of input image.");
-DEFINE_int32(input_width, 432, "Width of input image.");
+DEFINE_string(model_file, "../../data/models/hao28-600000-256x384.uff", "Path to uff model.");
+DEFINE_int32(input_height, 256, "Height of input image.");
+DEFINE_int32(input_width, 384, "Width of input image.");
 
 // profiling flags
 DEFINE_int32(repeat, 1, "Number of repeats.");
-DEFINE_int32(batch_size, 4, "Batch size.");
+DEFINE_int32(batch_size, 8, "Batch size.");
 DEFINE_int32(gauss_kernel_size, 17, "Gauss kernel size for smooth operation.");
 DEFINE_bool(use_f16, false, "Use float16.");
 DEFINE_bool(flip_rgb, true, "Flip RGB.");
 
 // input flags
-DEFINE_string(image_files, "", "Comma separated list of pathes to image.");
+DEFINE_string(
+        image_files,
+        "../../data/media/COCO_val2014_000000000192.jpg,"
+        "../../data/media/COCO_val2014_000000000459.jpg,"
+        "../../data/media/COCO_val2014_000000000415.jpg,"
+        "../../data/media/COCO_val2014_000000000564.jpg,"
+        "../../data/media/COCO_val2014_000000000294.jpg,"
+        "../../data/media/COCO_val2014_000000000623.jpg,"
+        "../../data/media/COCO_val2014_000000000357.jpg,"
+        "../../data/media/COCO_val2014_000000000488.jpg",
+        "Comma separated list of paths to image."
+        );
 
 int main(int argc, char *argv[])
 {
-    TRACE_SCOPE(__func__);
```

Comment:
I think `;` is still needed.

Reply:
Yep. I did not enable `TRACE` when cmaking, so I still passed the compilation.